main.nf

```diff
@@ -788,24 +1115,43 @@ process gvcf_combine {
 
 	output:
 		set group, id, file("${group}.combined.vcf"), file("${group}.combined.vcf.idx") into combined_vcf
+		set group, file("*versions.yml") into ch_gvcf_combine_versions
 
 	script:
 		all_gvcfs = vcf.join(' -v ')
 
+		"""
+		sentieon driver \\
+			-t ${task.cpus} \\
+			-r $genome_file \\
+			--algo GVCFtyper \\
+			-v $all_gvcfs ${group}.combined.vcf
+
+		${gvcf_combine_version(task)}
+		"""
+
+	stub:
+		"""
+		touch ${group}.combined.vcf
+		touch ${group}.combined.vcf.idx
+
+		${gvcf_combine_version(task)}
+		"""
+}
+def gvcf_combine_version(task) {
 	"""
-	sentieon driver \\
-		-t ${task.cpus} \\
-		-r $genome_file \\
-		--algo GVCFtyper \\
-		-v $all_gvcfs ${group}.combined.vcf
+	cat <<-END_VERSIONS > ${task.process}_versions.yml
+	${task.process}:
+	    sentieon: \$(echo \$(sentieon driver --version 2>&1) | sed -e "s/sentieon-genomics-//g")
+	END_VERSIONS
 	"""
 }
 
 // Create ped from input variables //
 process create_ped {
 	tag "$group"
 	time '20m'
-	publishDir "/${OUTDIR}/ped", mode: 'copy' , overwrite: 'true'	
+	publishDir "${OUTDIR}/ped", mode: 'copy' , overwrite: 'true'	
```

Comment:
Good catch. Were the files from this process just silently not published at all or did it somehow still work?

Reply:
It somehow worked, but you were obliged to use absolute paths (i.e. `/my/dir` would become `//my/dir` which also works).